### DISCUSSION TITLE
seems like the "global editing" is back?

### DISCUSSION THREAD

**Discussion by abuazizv** [YimMenu/YimMenu#3253] [General]
https://support.rockstargames.com/articles/30305516207635/GTA-Online-PC-Temporary-Change-FAQ

https://rockstarintel.com/gta-online-casino-heist-disabled-due-to-exploit/

https://gtaforums.com/topic/885020-psa-cheat-awareness-discussion-notices-news/page/212/#comment-1072391211

can someone explain of how true is this and whats really going on ? some people said its "global editing" again, a p2c still use something like that but it never really worked the feature called "i love rockstar" but it never really worked.

and does yimmenu prevent this ?

https://www.youtube.com/watch?v=dixDAkNwytU

on this video the guy says that some people wallpapers got changed and this sounds like an RCE exploit to me.

**Comment by Yimura** [comment]
YimMenu does not prevent this as we don't know what's happening ourselves really. If we don't know what's happening or how it works then there's no way in preventing it from happening.

No clue how "I love Rockstar" works either so I guess we don't properly protect against this, if it is an RCE then it's probably best not to go online for now. They seem to have temporarily fixed it by just disabling everything related to the casino??? Either way probably best to wait until further news as the announcement from R* was fairly recently posted.

**Reply by abuazizv** [reply]
iirc the "i love rockstar" do almost the same when some paid menus added the "remote" recovery where they could remove/add money to players etc...
